front/lib/actions/mcp_internal_actions/servers/ashby/rendering.ts

```diff
@@ -46,3 +47,89 @@ export function renderReportInfo(
     "The data has been saved as a CSV file."
   );
 }
+
+function renderSingleFeedback(feedback: AshbyFeedbackSubmission): string {
+  const lines = [];
+
+  if (feedback.submittedByUser) {
+    lines.push(
+      `**Submitted by:** ${feedback.submittedByUser.firstName} ${feedback.submittedByUser.lastName} (${feedback.submittedByUser.email})`
+    );
+  }
+
+  if (feedback.submittedAt) {
+    lines.push(
+      `**Submitted at:** ${new Date(feedback.submittedAt).toISOString()}`
+    );
+  }
+
+  lines.push("");
+
+  if (feedback.submittedValues && feedback.formDefinition.sections) {
+    for (const section of feedback.formDefinition.sections) {
+      for (const fieldWrapper of section.fields) {
+        const field = fieldWrapper.field;
+        const value = feedback.submittedValues[field.path];
+
+        if (value === undefined || value === null) {
+          continue;
+        }
+
+        lines.push(`**${field.title}:**`);
+
+        if (Array.isArray(value)) {
+          lines.push(value.join(", "));
+        } else if (
+          field.type === "ValueSelect" &&
+          field.selectableValues &&
+          typeof value === "string"
+        ) {
+          const selectedOption = field.selectableValues.find(
+            (opt) => opt.value === value
+          );
+          lines.push(selectedOption ? selectedOption.label : String(value));
+        } else if (typeof value === "object") {
+          lines.push(JSON.stringify(value, null, 2));
+        } else {
+          lines.push(String(value));
+        }
+
+        lines.push("");
+      }
+    }
+  }
+
+  return lines.join("\n");
+}
+
+export function renderInterviewFeedbackRecap(
```

Comment:
unless I missed it, I would  make sure the `applicationID` (and title/name) is in this list. Otherwise we could have the wrong one filled